runtime/libffi/z/sysvz64.s

```diff
@@ -1069,4 +953,4 @@ LSTOR DS  XL800
 SSTOR DS  XL32
 SSIZE DS  XL8
 DSASZ    EQU (*-CEEDSAHP_FIXED)
- END FFISYS
+  END FFISYS
```

Comment:
Either don't change this line or align `END` with `EQU` on the previous line.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/compare/583867825be4477a8319f21ffb952780631fbeb3..f6eb07952b7a7b7c8c8789fbb19e2c66f7672683